ui/src/components/visualizations/MapLibre/index.tsx

```diff
@@ -0,0 +1,220 @@
+import { MapPinAreaIcon } from "@phosphor-icons/react";
+import { Cross2Icon } from "@radix-ui/react-icons";
+import maplibregl, { LngLatBounds } from "maplibre-gl";
+import * as React from "react";
+import { useRef, useState, useMemo, useCallback } from "react";
+import { Map, Source, Marker, Layer, LayerProps } from "react-map-gl/maplibre";
+
+import "maplibre-gl/dist/maplibre-gl.css";
+
+import { Button, IconButton } from "@flow/components";
+import { useT } from "@flow/lib/i18n";
+import { SupportedDataTypes } from "@flow/utils/fetchAndReadGeoData";
```

Comment:
I'm surprised this isn't triggering a warning. I think css needs to be below all other imports